README.md

```diff
@@ -209,7 +209,8 @@ function cardExpiry(val) {
   const MAX_VAL = 1400;
   const withValueLimit = (inputObj) => {
     const { value } = inputObj;
-    if (value < MAX_VAL) return inputObj;
+    if (value >= MAX_VAL) return false;
```

Comment:
Nice catch! @alexanderdavide We can simplify this into something like this
```
const withValueLimit = inputObj => value <= MAX_VAL;
```
This should return either `true` or `false` and we may not need explicit return statements. Also, I guess it should be `value <= MAX_VAL` since `2 <= 2` is also a valid case 🤔

Reply:
Thank you for the recommendations. Happy to use shorthands, but initially opted for a more explained version. You're totally right with the less than sign. I mixed things up.

I have updated the PR. Furthermore, I think it's better to rely on `floatValue` because you have the exact value to do comparisons on. `value` is rounded; hence it omits the precision part of the value. Think about an actual value of 1400.4: It would be transferred as 1400 in `value` and pass the check which is a false positive. This can be prevented using `floatValue`.

Reply:
@alexandrapersea Sounds great! 

One suggestion here.. let's add both the examples.. one with `value` and one with `floatValue`. That way it will be very evident about what can be done and it's a good to have since it's always a good idea for documentation to be extensive.  Thanks again!

Reply:
I have added an explanation outlining the use cases. In my opinion, the text makes it very clear. I decided against adding code for both `value` and `floatValue` to not make it too verbose.

Reply:
@alexandrapersea Sounds good. That works too. Thanks!

Reply:
Great. Would you then undo the change request so that we can complete this PR?

Reply:
@alexanderdavide Done!